app/scripts/controllers/eventRegistrations.js

```diff
@@ -216,47 +216,25 @@ angular
           return 0;
         }
 
-        if (orderBy === 'completed_timestamp') {
-          return $scope.getRegistration(registrant.registrationId)
-            .completedTimestamp;
-        } else if (orderBy === 'created_timestamp') {
-          return registrant.createdTimestamp;
-        } else if (orderBy === 'registrant_type_id') {
-          return registrant.registrantTypeId;
-          //return $scope.getRegistrantType(registration.registrantTypeId).name;
-        } else if (orderBy === 'first_name') {
-          return registrant.firstName;
-        } else if (orderBy === 'last_name') {
-          return registrant.lastName;
-        } else if (orderBy === 'email') {
-          return registrant.email;
-        } else if (orderBy === 'checked_in_timestamp') {
-          return registrant.checkedInTimestamp;
-        } else if (orderBy === 'last_updated_timestamp') {
-          return registrant.lastUpdatedTimestamp;
-        } else if (orderBy === 'group_id') {
-          return registrant.groupId;
-        } else {
-          var answerValue = findAnswer(registrant, orderBy);
-          if (angular.isUndefined(answerValue)) {
-            return '';
-          }
+        var answerValue = findAnswer(registrant, orderBy);
```

Comment:
I removed all of this as it was old filtering logic that the frontend was doing before API filtering was introduced. It was actually conflicting with the sorting that the API was doing.